tests/acceptance/test_config_refresh.py

```diff
@@ -73,3 +74,21 @@ def test_no_config_refresh_after_5_seconds(tmp_path, config):
         "Configuration version didn't change. Continue running with current version.",
         test_timeout=7,
     )
+
+
+def test_recover_after_invalid_then_valid_config(tmp_path, config):
+    config_path = tmp_path / "generated_config.yml"
+    config_path.write_text(config.as_json())
+    proc = start_logprep(config_path)
+    wait_for_output(proc, "Config refresh interval is set to: 5 seconds", test_timeout=5)
+
+    config.config_refresh_interval = None
+    config.version = "invalid"
```

Comment:
as described in https://github.com/fkie-cad/Logprep/blob/00688049ef280e41bd4ece256eddcf70d459b7bc/logprep/util/configuration.py#L434-L435 the version  attribute is of type "str". So setting it to the string "invalid" doesn't result in an invalid configuration.